tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Replace this cell by:</p><p>"Notice we are now using 100 runs in order to see the effects of the modulations clearly."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>